cmd/deploy/CortexClient.go

```diff
@@ -413,6 +416,77 @@ func DeployCampaign(cortex CortexClientV6, filename string, deployable bool, ove
 	return err
 }
 
+func DeployModel(cortex CortexClientV6, filepath string) string {
+	content, err := GetJsonContent(filepath)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	model := gjson.Parse(string(content))
+	status := model.Get("status").String()
+	// models can't be created with Published status, but exported only if published. So creating without status (using default initial status) and saving again with Published status
+	if status == "Published" {
+		modelBody := model.Value().(map[string]interface{})
+		modelBody["status"] = "In development"
+		initial, _ := json.Marshal(modelBody)
+		res, err := post(&cortex, "/fabric/v4/projects/"+cortex.Project+"/models", initial)
+		if err != nil {
+			log.Println(string(res))
+			log.Fatalln(err)
+		}
+	}
+	res, err := post(&cortex, "/fabric/v4/projects/"+cortex.Project+"/models", content)
+	if err != nil {
+		log.Println(string(res))
+		log.Fatalln(err)
+	}
+	return string(res)
+}
+
+func DeployExperiment(cortex CortexClientV6, filepath string) string {
+	content, err := GetJsonContent(filepath)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	res, err := post(&cortex, "/fabric/v4/projects/"+cortex.Project+"/experiments", content)
```

Comment:
maybe store this fragment in a string whenever you can so that if the `v4` bit changes at some point, we can update everything in a single line change

Reply:
inconsistent spacing for string concats.. run this through `go fmt` https://go.dev/blog/gofmt

Reply:
you can setup your code editor to run `go fmt` on every save

Reply:
gofmt adds space depending on context https://github.com/golang/go/issues/1206. Whenever string concat is in params of a func call, space is omitted. But in variable assignment its added

Tried to do manually, but on save this plugin overrides